plone/app/discussion/upgrades.zcml

```diff
@@ -50,5 +50,15 @@
         handler=".upgrades.upgrade_comment_workflows"
         />
   </genericsetup:upgradeSteps>
+  
+  <genericsetup:upgradeStep
+      title="show email in moderation view"
+      description="reload catalog config to add author_email in metadata.'"
+      source="1000"
+      destination="1001"
+      handler=".upgrades.update_catalog"
+      sortkey="1"
+      profile="plone.app.discussion:default"
+      />
```

Comment:
Usually core packages should have their upgrade steps on plone.app.upgrade, so whenever a Plone installation is updated, one only needs to care about the upgrades on p.a.upgrade.

p.a.discussion started as an add-on, so that's why it has its own upgrades, I would prefer if we move this upgrade step to plone.app.upgrade and on another pull request we clean the upgrades that are still in this repository.